### ISSUE TITLE
Column headers width is wrong if responsive is disabled and bulk actions are enabled [Bug] 

### ISSUE THREAD

**Issue by alfonsocuccaro** [Laravel-Backpack/CRUD#3138]
# Bug report

### What I did

I have enabled on a CrudController bulk delete and `called $this->crud->disableResponsiveTable();` . Without the bulk operations enabled (the first column checkbox visible) it works fine.

### What I expected to happen

To normally see the table without the responsive management

### What happened

Visually the table header moved as you can see in the picture 
![bug-backpack](https://user-images.githubusercontent.com/6061785/90881966-61538e00-e3ab-11ea-8739-e76a411fc8db.png)

it also happens in desktop view 
![image](https://user-images.githubusercontent.com/6061785/90882330-17b77300-e3ac-11ea-92a3-399ebd61c9c3.png)

### What I've already tried to fix it

I have made some tests and it seems to happen because of the `scrollX: true,` inside `vendor\backpack\crud\src\resources\views\crud\inc\datatables_logic.blade.php` . If I force it to be false it doesn't happen, but of course the scroll x doesn't work

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

### PHP VERSION:
PHP 7.4.4 (cli) (built: Mar 17 2020 13:49:19) ( ZTS Visual C++ 2017 x64 )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies

### LARAVEL VERSION:
v7.18.0@116b508bafd81de97b1c09744445d32a91076b60

### BACKPACK VERSION:
4.1.12@e5e3efc89775574fff03ef0000677e9a85903c9b

**Comment by alfonsocuccaro** [comment]
If anyone is in this situation I found a workaround:

1. Download the [datatables_logic.blade.zip](https://github.com/Laravel-Backpack/CRUD/files/5109260/datatables_logic.blade.zip) 
2. Extract it
3. Put the ```datatables_logic.blade.php``` file in ```/your-project/resources/views/vendor/backpack/crud/inc``` (if you don't have this directory create it)
4. Put in your  ```/your-project/resources/sass/app.scss```file this code 
```
.momentum-x{
  overflow-x: auto;
}
```
5. run `npm run production`

**DISCLAIMER:** This is only a workaround, I think it will come a better option

**Comment by pxpm** [comment]
Hello @alfonsocuccaro 

is it possible that this had been fixed with: https://github.com/Laravel-Backpack/CRUD/pull/3157

and you are not using the last available version of the file ? 

Check your `public/packages/backpack/crud/css/list.css` for the changes made in that pull request if you already have them. 

Let me know,
Pedro

**Comment by pxpm** [comment]
I'v tested this solution, the problem is that using `scrollX: false` and adding `overflow-x` will make the scroll bar appear on bottom of the table, and not before the table footer, thus you would scroll whole table container hidding buttons etc while scrolling.

From my tests, the solution provided by @promatik would also not suffice. 

I'v submitted PR #3190 

Can you please test my solution and give your feedback ? 

Thanks in advance.

Best,
Pedro

**Comment by alfonsocuccaro** [comment]
Hi and sorry for the delay.
I have tested it and with a quick look it seemed ok, so I approved the
change, but then I looked more closely and noticed that the columns
alignment wasn't ok yet, so I commented.

Let me know if I can do more, I'll be faster.

Thank you

*Alfonso Cuccaro*

Responsabile produzione

*Tel:*  0815706309
*E-mail:* alfonso.cuccaro@webapp.it
*Web:* www.webapp.it
*Skype:* alfonso.cuccaro

<http://www.webapp.it/>
*WEBAPP *S.r.l. - Contrada Pisciarelli N. 9 - 80078 Pozzuoli (NA) - ITALY
<https://maps.google.com/?q=Contrada+Pisciarelli+N.+9+-+80078+Pozzuoli+(NA)+-+ITALY&entry=gmail&source=g>

Fax 06 23314997  p.iva 06665281215

AVVISO DI RISERVATEZZA

Il testo e gli eventuali documenti trasmessi contengono informazioni
riservate al destinatario indicato.

La seguente e-mail è confidenziale e la sua riservatezza è tutelata
legalmente dal codice della privacy art.13 del D.Lgl. 30.6.2003 n°196. La
lettura, copia od altro uso non autorizzato o qualsiasi altra azione
derivante dalla conoscenza di queste informazioni sono rigorosamente
vietate. Se pensi di non essere il destinatario di questa mail, o se hai
ricevuto questa mail per errore, sei pregato di darne immediata
comunicazione al mittente e di provvedere immediatamente alla sua
distruzione.
Questo messaggio rappresenta l'opinione del mittente e non necessariamente
l'opinione dell'azienda.



Il giorno mar 8 set 2020 alle ore 14:01 Pedro Martins <
notifications@github.com> ha scritto:
